backend/bots/consumers.py

```
@@ -110,21 +120,20 @@ async def connect(self):
 			await self.make_bot_move()
 
 	async def make_bot_move(self):
-		
+		bot_move_process_start = time.perf_counter()
```

Ruff (0.8.2)

Local variable `bot_move_process_start` is assigned to but never used

Remove assignment to unused variable `bot_move_process_start`

(F841)